src/frontend/tsx/GamePanel.tsx

```diff
@@ -60,7 +60,7 @@ export function GamePanel () {
             </span>
             { '\xa0'.repeat(6 - wMove.length) }
             {/* setting opacity to 1.0 offsets a cumulative reduction in opacity on each bIndex ply when displayIndex < this move's wIndex */}
-            <span onClick={ () => setDisplayIndex(bIndex) } style={{ opacity: (moveOpacity(wIndex) == 1.0) ? moveOpacity(bIndex) : 1.0 }}>
+            <span onClick={ () => setDisplayIndex(bIndex) } style={{ opacity: (moveOpacity(wIndex) === 1.0) ? moveOpacity(bIndex) : 1.0 }}>
```

Comment:
@bonbud-macryg is this alright?
the linter complained about this not being strict equivalence. so I added an `=`.

Reply:
I think I changed it from `===` to `==` because *my* linter complained about it. Whichever version of this doesn't get complained about by Docker should be considered the correct one.